PythonCodeConventions.md

```diff
@@ -1,25 +1,33 @@
 Python Code Conventions
 =======
 
-PEP8 is the code style guide for Python code used by the main python distribution.
-This style guide can be found at:
-https://www.python.org/dev/peps/pep-0008/
+General
+---
```

Comment:
This document is mixing two markdown styles for the headers. Please make whole document consistent to use the newer style wit h'#', '##', etc.

Reply:
Will do